packages/kbn-unified-data-table/src/components/custom_control_columns/index.ts

```diff
@@ -12,3 +12,9 @@ export {
 } from './color_indicator';
 
 export { getAdditionalRowControlColumns } from './additional_row_control';
+
+/**
+ * Preset controls
+ */
+export { createDegradedDocsControl } from './degraded_docs_control';
+export { createStacktraceControl } from './stacktrace_control';
```

Comment:
Thanks for extracting the code! One suggestion is to move the logs controls from the generic `UnifiedDataTable` to discover-utils package for example.

Reply:
I thought the same initially, then I realized we would need to use the row control related types, currently defined in the UnifiedDataTable package, into `discover-utils`, which would create a cyclic dependency between the 2 packages.
I could move also the related types into `discover-utils` and let the unified data table package consume them to prevent this scenario, would that look good? Otherwise leaving this as it is now would be the other option to prevent the dependency issue, wdyt? I'm good to go with both options

Reply:
Moving type definitions to discover-utils sounds good to me. We can still reexport them from `UnifiedDataTable` types file.